rebasehelper/cli.py

```diff
@@ -143,6 +150,13 @@ def add_args(self):
             help="Specify number of retries in case build fails.",
             type=int
         )
+        self.parser.add_argument(
+            "--enable-option",
+            dest="enable_option",
```

Comment:
this argument should not be needed, the value should be placed into variable named the same way by default.